concepts/0250-rich-schemas/README.md

```diff
@@ -0,0 +1,360 @@
+credential attribute presented to the verifier.
+
+### Presentation Definitions
+A presentation definition is the means whereby a verifier asks for data
+from a holder. It contains a set of named desired proof attributes with
+corresponding restrictions that limit the potential sources for the
+attribute data according to the desired source schema, issuer DID,
+credential definition, etc. A presentation definition also contains a
+similar set of requested predicate proofs, with named attributes and
+restrictions.
+
+It may be helpful to think of a presentation definition as the
+mirror image of a mapping object. Where a mapping object specifies the
+graph paths of the attributes to be signed, a presentation definition
+specifies the graph query that may be fulfilled by such graph paths. The
+presentation definition does not need to concern itself with specifying a
+particular mapping that contains the desired graph paths, any mapping that
+contains those graph paths may be acceptable. The fact that multiple graph
+paths might satisfy the query adds some complexity to the presentation
+definition. The query may also restrict the acceptable set of issuers and
+credential definitions and specify the desired predicates.
+
+A presentation definition is expressed using JSON-LD and may be stored in a
+data registry. This supports re-use, interoperability, and a much
+richer set of communication options. Multiple verifiers can use the same
+presentation definitions. A community may specify acceptable presentation
+definitions for its verifiers, and this acceptable set may be adopted by
+other communities. Credential offers may include the presentation
+definition the issuer would like fulfilled by the holder before issuing
+them a credential. Presentation requests may also be more simply negotiated
+by pointing to alternative acceptable presentation definitions. Writing a
+presentation definition to a data ragistry also allows it to be publicly
+reviewed for privacy and security considerations and gain or lose
+reputation.
+
+### Presentations
+The presentation object that makes use of rich schemas is defined by the
+W3C Verifiable Credentials Data Model, and is known in the specification as
+a verifiable presentation. The verifiable presentation is defined as a way
+to present multiple credentials to a verifier in a single package.
+
+The claims that make up a presentation are specified by the presentation
+definition. The credentials from which these claims originate are used to
+create new derived credentials that only contain the specified claims and
+the cryptographic material necessary for proofs. The type of claims in
+derived credentials is also specified by the presentation definition. These
+types include revealed and predicate proof claims. The presentation also
+contains the cryptographic material which supports a proof that source
+credentials are held by the same entity. This is accomplished by proving
+knowledge of a link secret.
```

Comment:
I am curious about the linking between https://w3c.github.io/vc-data-model/#zero-knowledge-proofs and this RFC?

I'm also wondering if the anonymous credential material (e.g., link secrets) should be coupled with the rich schema material?